### DISCUSSION TITLE
Reactivity example not workking

### DISCUSSION THREAD

**Discussion by derekdai** [HapticX/happyx#212] [General]
I can't get this reactivity example work
```nim
import happyx

var x = remember 0

appRoutes "app":
  "/":
    tDiv:
      "x counter: {x}"
      @click:
        x->inc()
        console.log(x)
```

Although event handler can be triggered by click, the corresponding elements will not be updated. Have I done anything wrong?

**Comment by Ethosa** [comment]
This seems like a bug .. 🤔

**Comment by Ethosa** [comment]
solved in #213
